tests/python_tests/utils/ov_genai_pipelines.py

```diff
@@ -0,0 +1,57 @@
+# Copyright (C) 2018-2025 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+from enum import Enum
+from pathlib import Path
+
+from openvino_genai import SchedulerConfig, draft_model, ContinuousBatchingPipeline, LLMPipeline
+
+def dict_to_scheduler_config(scheduler_params: dict = None) -> SchedulerConfig:
+    scheduler_config = SchedulerConfig()
+    if scheduler_params is None:
+        scheduler_config.dynamic_split_fuse = True
+        # vLLM specific
+        scheduler_config.max_num_batched_tokens = 256
+        scheduler_config.max_num_seqs = 256
```

Comment:
Let's create a task to create `SchedulerConfig` from map on `pybind11` level?

Reply:
I will create tasks to improve test infra, will contain this ticket also